File: jsfring-webservice/src/main/java/fr/pinguet62/jsfring/webservice/dto/UserDto.java
package fr.pinguet62.jsfring.webservice.dto;

import fr.pinguet62.jsfring.model.sql.User;
import lombok.Data;

import java.time.LocalDateTime;
import java.util.Set;

/**
 * @see User
 */
@Data
public final class UserDto {

    private Boolean active;

    private String email;

    private LocalDateTime lastConnection;

    private Set<Integer> profiles;

}

File: jsfring-webservice/src/main/kotlin/fr/pinguet62/jsfring/webservice/dto/UserDto.kt
package fr.pinguet62.jsfring.webservice.dto

import fr.pinguet62.jsfring.model.sql.User
import java.time.LocalDateTime

/**
 * @see User
 */
data class UserDto(
        var active: Boolean?,
        var email: String?,
        var lastConnection: LocalDateTime?,
        var profiles: Set<Int>?
) {
    constructor() : this(null, null, null, null)
}
